elijah/README.md

```diff
@@ -0,0 +1,228 @@
+### Homework #1
+(Due Monday 7/20)
+
+You need to turn in this homework by cloning this repo, making a new branch with your solutions, pushing to github, and issuing a pull request.
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** (_30% of total time_)
+Predict the output of each of the following expressions, and explain the rules which determine that answer.  (_Hint: The rules for operator `==` are complicated, but do your best!_)
+
+If the output depends on the value or type of variable _x_, identify the conditions (what _x_ is) when the expression will output true (or false, if that's simpler).  Assume the cases are independent, and _x_ is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.
+
+
+**a)** `"1" == 1`
+    true: js will convert right hand side to string
+
+**b)** `"1" === 1`
+    false: strict equal does no type conversion
+
+**c)** `x == 'x'`
+    false: if x is set to anything other than 'x'
+
+**d)** `x == (x+'')`
+    false if x is undefined: evaluates `x+''` to string 'undefined'
+    true if x is set: ignores the empty string
+
+**e)** `'' == ' '`
+    false: empty string is 'falsey'
+
+**f)** `x = true`
+    true: sets the value of x to true
+
+**g)** `var x; x == 'undefined'`
+    false: x is primitive undefined, right side is a string
+
+**h)** `'9'<'10'`
+    false: comparison of strings is done character by character
+
+**i)** `typeof x + 1 === "number"`
+    false: evaluates `typeof x` as 'undefined', then adds 1 as a string making
+    'undefined1'
+
+**j)** `typeof x % 2 === "number"`
+    false: evaluates `typeof x` as 'undefined', 'undefined'%2 is NaN
+
+**k)** `typeof (x % 2) === "number"`
+    true: evaluates `x%2` as NaN, typeof NaN is 'number'
+
+**l)** `x++ == ++x`
+    false: either side returns NaN, NaN != NaN
+
+**m)** `++x == x++`
+    false: either side returns NaN, NaN != NaN
+
+**n)** `"1"+x == 1+x`
+    false: type coersion makes left side "1undefined". right side is NaN
+
```

Comment:
Similarly, you are correct but there can be exceptions to this. If x is a string, coercion will take place and they will be similar to one another.